CHANGELOG.md

```diff
@@ -1,22 +1,36 @@
 # astro changelog
 
-## 0.5.0 (UNRELEASED, 2019)
+## 0.5.0 (October 3, 2019)
 
+### Added
+* Add ls command to tvm (#37)
 * Add `version` command (#12)
 * Add binaries via goreleaser (#14)
+* Support detach flag with Terraform 0.12 (#45)
+* Make plan output work with terraform 0.12 (#41)
+* Add Travis configuration, `make lint` and git precommit hook
+
+### Changed
+* Don't pass variables to the modules that don't declare them (#40)
 * Adopt options pattern for `astro.NewProject` constructor (#26)
 * Refactor and improve integration tests to invoke them directly using cli
   rather than `os.exec` (#26)
-* Add Travis configuration, `make lint` and git precommit hook
+* Remove godep and move to Go modules (vgo)
+* Change configuration syntax for remapping CLI flags to Terraform module
+  variables
+
+### Security
+* Fix zip slip vulnerability (#47)
```

Comment:
`* Fix zip slip vulnerability in tvm (#47)`

Reply:
Done.